File: tools/src/main/java/cn/pandadb/tool/UnsafePNodeLauncher.java
package cn.pandadb.tool;

/**
 * @Author: Airzihao
 * @Description:
 * @Date: Created at 19:52 2019/12/25
 * @Modified By:
 */
public class UnsafePNodeLauncher {

    public static void main(String[] args) {
        String num = args[0];
        String dbPathStr = "../itest/output/testdb/db" + num;
        String confFilePathStr = "../itest/testdata/localnode" + num + ".conf";
        String[] startArgs = {dbPathStr, confFilePathStr};
        PNodeServerStarter.main(startArgs);
    }
}

File: tools/src/main/scala/cn/pandadb/tool/UnsafePNodeLauncher.scala
package cn.pandadb.tool

/**
  * @Author: Airzihao
  * @Description:
  * @Date: Created in 22:27 2019/12/25
  * @Modified By:
  */
object UnsafePNodeLauncher {
  def main(args: Array[String]): Unit = {
    val num = args(0)
    PNodeServerStarter.main(Array(s"./itest/output/testdb/db${num}",
      s"./itest/testdata/localnode${num}.conf"))
  }
}
